index.js

```diff
@@ -10,11 +10,25 @@ const util = require('util')
 const mc = require('minecraft-protocol')
 const debug = require('debug')('chunk-dumper')
 
+function flattenMaskArray (sectionBitMask) {
```

Comment:
this is not needed here and introduces boilerplate everywhere if used
prismarine-chunk needs to handle all transformations